src/main/java/com/taskadapter/redmineapi/bean/GenericAssigneeFactory.java

```diff
@@ -0,0 +1,8 @@
+
+package com.taskadapter.redmineapi.bean;
+
+public class GenericAssigneeFactory {
```

Comment:
I originally went with the idea of those factory classes assuming that we will have more complex constructors.
but if it's just id and name and all the factory does is calling a constructor with the provided params, then I would just use the constructor of GenericAssignee directly